source/PlayniteUI/Windows/FirstTimeStartupWindow.xaml

```diff
@@ -12,7 +12,7 @@
         Style="{DynamicResource StandardWindowStyle}"
         Title="{DynamicResource LOCFirstWindowTitle}" WindowStartupLocation="CenterScreen"
         ShowCloseButton="False" ShowMaximizeButton="False" ShowMinimizeButton="False"
-        Height="450" Width="550" x:Name="WindowMain"

```

Comment:
This makes dialog super huge, please change this to some default value.

Reply:
the height and width definition is in the theme now because it is dependent on the theme.  If the setting is here the theme setting gets overridden.  If there a way like in CSS to force the theme to override I can do that.

Reply:
Yes, moved the sizing back to the form (using the largest of the 2 so both styles work nice by default).  Wasn't thinking it being the "default" fallback for other windows, though they all have explicit sizes set right now and it wasn't applying to them.